debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerStreamingChangeEventSource.java

```diff
@@ -201,20 +314,24 @@ public int getOperation() throws SQLException {
         public boolean next() throws SQLException {
             completed = !resultSet.next();
             currentChangeLsn = completed ? Lsn.NULL : Lsn.valueOf(resultSet.getBytes(COL_ROW_LSN));
+            if (completed) {
+                LOGGER.trace("Closing result set of change table for table {}", changeTable);
```

Comment:
s/change table for table {}/change table {}/? Seems you don't have the original table name handy.